mpet/props_elyte.py

```diff
@@ -147,3 +147,79 @@ def sigma_ndim(c):
         return sigma(c) * (
             k*Tref/(e**2*Dref*N_A*(1000*cref)))
     return D_ndim, sigma_ndim, therm_fac, tp0, Dref
+
+
+
+def LIONSIMBA():
+    """ Set of parameters from LIONSIMBA validation. Torchio et al, 2016.
+    """
+    T = 298 #isothermal model
+
+    def tp0(c):
+        return 0.364
+
+    def sigma(c):
+        c_dim = c*1000 #dimensionalized c
+        r1 = -10.5
+        r2 = 0.668e-3
+        r3 = 0.494e-6
+        r4 = 0.074 
+        r5 = -1.78e-5
+        r6 = -8.86e-10
+        r7 = -6.96e-5
+        r8 = 2.8e-8
+        sig_out = 1e-4 * c_dim * (r1 + r2*c_dim + r3*c_dim**2 + T*(r4 + r5*c_dim + r6*c_dim**2) + T**2 *(r7 + r8*c_dim))**2
+        return sig_out  # m^2/s
+
+    def D(c):
+        c_dim = c*1000
+        T = 298
+        r1 = 4.43
+        r2 = 54
+        r3 = 229
+        r4 = 5e-3
+        r5 = 0.22e-3
+        D_out = 1e-4 * 10**(-r1-r2/(T-r3-r4*c_dim)-r5*c_dim)
+        return D_out
+
+    def therm_fac(c):
+        return 1.
+
+    Dref = D(cref)
+
+    def D_ndim(c):
+        return D(c) / Dref
+
+    def sigma_ndim(c):
+        return sigma(c) * (
+            k*Tref/(e**2*Dref*N_A*(1000*cref)))
+    return D_ndim, sigma_ndim, therm_fac, tp0, Dref
+
+
+def LIONSIMBA_isothermal():
+    """ Set of parameters from LIONSIMBA validation. Torchio et al, 2016.
+    """
+    T = 298 #isothermal model
+
+    def tp0(c):
+        return 0.364
+
+    def sigma(c):
+        ce = c*1000 #dimensionalized c
+        return 4.1253e-2 + 5.007e-4*ce - 4.7212e-7*ce**2 +1.5094e-10*ce**3 -1.6018*1e-14*ce**4 # S/m
```

Comment:
I think I recognize the boltzmann constant, but just wondering if some of these values should be defined as variables just for clarity?

Reply:
most constants are just fitted values from the LIONSIMBA model :( so kind of difficult to explain

Reply:
Unfortunately, many battery models fit very complicated functions to data,and the individual terms have no particular meaning. I like the idea of moving toward lookup tables for storing electrolyte properties and OCV.